plugin/src/main/kotlin/com/varabyte/kobweb/intellij/notification/KobwebNotifier.kt

```diff
@@ -0,0 +1,134 @@
+package com.varabyte.kobweb.intellij.notification
+
+import com.intellij.notification.Notification
+import com.intellij.notification.NotificationAction
+import com.intellij.notification.NotificationGroupManager
+import com.intellij.notification.NotificationType
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.project.Project
+
+private val notificationGroup by lazy {
+    NotificationGroupManager.getInstance().getNotificationGroup("Kobweb")
+}
+
+/**
+ * A handle to a notification that provides some constrained operations on it.
+ */
+class KobwebNotificationHandle(private val notification: Notification) {
```

Comment:
We might have found ourselves a use case for value classes! 🚀🚀🚀

Reply:
Done :)